demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -425,6 +425,22 @@ svg {
   }
 }
 
+.video-preview {
```

Comment:
1. You can put this mobile view into the existing mobile `@media` block at line 361.
    ```
    @media (max-width: 575px) { 
        .video-preview { ... }
    }
    ```